ledger/participant-integration-api/src/main/scala/platform/configuration/CommandConfiguration.scala

```diff
@@ -41,6 +41,6 @@ object CommandConfiguration {
       maxParallelSubmissions = 512,
       maxCommandsInFlight = 256,
       limitMaxCommandsInFlight = true,
-      retentionPeriod = 24.hours,
+      retentionPeriod = 5.minutes,
```

Comment:
Perhaps pull it out into a constant here, and use it in `kvutils/app/Config.scala`.

Reply:
Fixed.